lib/Pod/To/HTML.pm

```diff
@@ -1,5 +1,6 @@
 unit class Pod::To::HTML;
 use URI::Escape;
+use Template::Mustache;
```

Comment:
Doesn't look like `Template::Mustache` was added to `"depends"` section in `META6.json`

Reply:
Fixing. Travis also complained.